Package/PackageSchema.xsd

```diff
@@ -130,4 +130,24 @@
       <xs:attribute name="Tags" type="xs:string"/>
     </xs:complexType>
   </xs:element>
+
+  <xs:complexType name="Validation" abstract="true"/>
+
+  <xs:complexType name="FileExists">
+    <xs:complexContent>
+      <xs:extension base="Validation">
+        <xs:attribute name="Path" type="xs:string" use="required" />
+      </xs:extension>
+    </xs:complexContent>
+  </xs:complexType>
+
+  <xs:element name="Validation">
+    <xs:complexType>
+    <xs:sequence>
+      <xs:choice minOccurs="0" maxOccurs="unbounded">
+        <xs:element name="FileExists" type="FileExists"/>
+      </xs:choice>
+    </xs:sequence>
+    </xs:complexType>
+  </xs:element>
```

Comment:
Will this schema change make older versions of OpenTAP unable to parse the content of new OpenTAP packages using this feature?

Reply:
This should not be a problem. At least old versions of KS8400 can list the packages just fine.